mappings/net/minecraft/client/gui/hud/debug/DebugHud.mapping

```diff
@@ -1,49 +1,85 @@
 	FIELD f_demcwxlr client Lnet/minecraft/unmapped/C_ayfeobid;
 	FIELD f_fjhfdzhz HEIGHT_MAP_TYPES Ljava/util/Map;
 	FIELD f_hdjainem fluidHit Lnet/minecraft/unmapped/C_eetoegzn;
+	FIELD f_jwptaljh showingCharts Z
+	FIELD f_kggcezmz frameRateChartRenderer Lnet/minecraft/unmapped/C_vgifhjob;
+	FIELD f_llewhquv tickRateSampleLog Lnet/minecraft/unmapped/C_fnzofwlw;
+	FIELD f_lmmlpwdr pingChartRenderer Lnet/minecraft/unmapped/C_ftwidtyt;
+	FIELD f_mriubqfx tickRateChartRenderer Lnet/minecraft/unmapped/C_rggqladj;
+	FIELD f_nggbpvpt bandwidthUsageChartRenderer Lnet/minecraft/unmapped/C_zjhfcbxz;
 	FIELD f_nivhpjpa TEXT_COLOR I
 	FIELD f_qympybag allocationRateCalculator Lnet/minecraft/unmapped/C_epxplegp$C_itbnpmom;
+	FIELD f_rhtdusnd bandwidthUsageSampleLog Lnet/minecraft/unmapped/C_fnzofwlw;
+	FIELD f_rlumovzv showProfilerChart Z
 	FIELD f_rmhsdqts MARGIN_LEFT I
 	FIELD f_tekegaef MARGIN_RIGHT I
 	FIELD f_unrhvpxp MARGIN_TOP I
 	FIELD f_uzvvfrgc pos Lnet/minecraft/unmapped/C_ynrszrtu;
 	FIELD f_vbjupvgy blockHit Lnet/minecraft/unmapped/C_eetoegzn;
 	FIELD f_vmhafjuy chunk Lnet/minecraft/unmapped/C_hrdsvlkq;
 	FIELD f_warxjbak chunkFuture Ljava/util/concurrent/CompletableFuture;
+	FIELD f_xemytmmw showingFpsTpsCharts Z
+	FIELD f_ydlzbbxe showNetworkCharts Z
 	FIELD f_yryzioge textRenderer Lnet/minecraft/unmapped/C_mavozmpp;
 	METHOD <init> (Lnet/minecraft/unmapped/C_ayfeobid;)V
 		ARG 1 client
+	METHOD m_asllhmxy toggleProfilerChart ()V
 	METHOD m_ayfaebbc renderLeftText (Lnet/minecraft/unmapped/C_sedilmty;)V
 		ARG 1 graphics
 	METHOD m_bwbghmzt renderText (Lnet/minecraft/unmapped/C_sedilmty;Ljava/util/List;Z)V
 		ARG 2 lines
 		ARG 3 leftSide
+	METHOD m_cfxfrpvw (Lcom/mojang/datafixers/util/Either;)Lnet/minecraft/unmapped/C_hrdsvlkq;
+		ARG 0 chunk
+	METHOD m_dfwofowe pushTickRateSample (J)V
+		ARG 1 sample
+	METHOD m_dqqzkibu toggleNetworkCharts ()V
 	METHOD m_enweuogg getServerWorld ()Lnet/minecraft/unmapped/C_bdwnwhiu;
 	METHOD m_eouaifzj (Lit/unimi/dsi/fastutil/objects/Object2IntMap;Lnet/minecraft/unmapped/C_ormqdxci;)Ljava/lang/String;
 		ARG 1 group
 	METHOD m_fxsledwt renderRightText (Lnet/minecraft/unmapped/C_sedilmty;)V
 		ARG 1 graphics
 	METHOD m_gdvpeupa getClientChunk ()Lnet/minecraft/unmapped/C_hrdsvlkq;
 	METHOD m_iucxvuxr getChunk ()Lnet/minecraft/unmapped/C_hrdsvlkq;
+	METHOD m_jedplmqj getPingSampleLog ()Lnet/minecraft/unmapped/C_fnzofwlw;
 	METHOD m_jfnssvmz toMiB (J)J
 		ARG 0 bytes
 	METHOD m_jlxirjjv resetChunk ()V
+	METHOD m_jlyaywtw reset ()V
+	METHOD m_kaxtdcsl getBandwidthUsageSampleLog ()Lnet/minecraft/unmapped/C_fnzofwlw;
+	METHOD m_kkowleuv networkChartsVisible ()Z
+	METHOD m_kzkpfpca chartsVisible ()Z
 	METHOD m_lausyuql getLeftText ()Ljava/util/List;
+	METHOD m_lqfjmxvr (Lnet/minecraft/unmapped/C_gkderdjo$C_mnhcarhk;)Lnet/minecraft/unmapped/C_hrdsvlkq;
+		ARG 0 unloaded
 	METHOD m_melioxxl getWorld ()Lnet/minecraft/unmapped/C_cdctfzbn;
+	METHOD m_mpdpmccw toggleFpsTpsCharts ()V
```

Comment:
should match `showingFpsTpsCharts` if it's changed

Reply:
This toggles it, rather than returning it.  So if it is `true` it becomes `false`, if it is `false` it becomes `true`.

Reply:
I meant the `FpsTps` part, so:
```suggestion
	METHOD m_mpdpmccw toggleRateCharts ()V
```

Reply:
Ah yeah sorry, forgot about that.